src/XrdHttp/XrdHttpReq.hh

```diff
@@ -164,18 +164,9 @@ private:
 
 public:
   XrdHttpReq(XrdHttpProtocol *protinstance, const XrdHttpReadRangeHandler::Configuration &rcfg) :
-      readRangeHandler(rcfg), keepalive(true) {
-
-    prot = protinstance;
-    length = 0;
-    //xmlbody = 0;
-    depth = 0;
-    opaque = 0;
-    writtenbytes = 0;
-    fopened = false;
-    headerok = false;
-    mScitag = -1;
-  };
+    prot(protinstance),
+    readRangeHandler(rcfg)
+  {}
```

Comment:
Not sure if others agree, but I find the old way easier to understand. I'd do
```suggestion
    prot(protinstance),
    readRangeHandler(rcfg),
    length(0),
    depth(0),
    opaque(nullptr),
    writtenbytes(0),
    fopened(false),
    headerok(false),
    mScitag(-1)
  {}
```
and drop the changes on the members directly.

Reply:
+1 yes, it's easier to see to what values the variables are assigned to by default in the constructor.

Reply:
> Regardless, I think either my or Guilherme's approach is reasonable -- the original code version of having it as statements in the constructor body is not.

I agree that the statements inside the constructor are not nice. They make it [non-trivial](https://en.cppreference.com/w/cpp/language/default_constructor#Trivial_default_constructor), for instance, which we'd like to avoid.